pom.xml

```diff
@@ -208,6 +208,19 @@
                 </executions>
             </plugin>
 
+            <plugin>
```

Comment:
Any particular reason for this change? `maven-source-plugin` is pulled in through the parent pom and executed on release builds.